### ISSUE TITLE
ClockManagement's clocks[n] is not driven on Xilinx parts - though ClockManagement's clocksout(n) is.

### ISSUE THREAD

**Issue by NickShaffner** [cfelton/rhea#50]
ClockManagement's clocks[n] is not driven on Xilinx parts - though ClockManagement's clocksout(n) is.

The title says it all, I have reproduced this on both Zybo and CMod_A7 device_primitives tests.

*** For Zybo:

in zybo_device_primitives.py change
```python
    clock1 = clkmgmt.clocksout(0)
```

to:
```python
   clock1 = clkmgmt.clocks[0]
```

and then do a:

`python zybo_device_primitives.py --build`

and flash the device.  Note that the corresponding led stays unlit.

This has also been verified to fail on the cmoda7 hardware by making a similar change.

**Comment by NickShaffner** [comment]
This is fixed in pull request https://github.com/cfelton/rhea/pull/49  - commit https://github.com/cfelton/rhea/pull/49/commits/570c138c09f70b29ee545eb918c6dd0b231bff4d

**Comment by NickShaffner** [comment]
Fixed by @cfelton's merge of pull request #49
